azurerm/internal/services/powerbidedicated/registration.go

```diff
@@ -0,0 +1,23 @@
+package powerbidedicated
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type Registration struct{}
+
+// Name is the name of this Service
+func (r Registration) Name() string {
+	return "PowerBIDedicated"
```

Comment:
same q here, should this be something else?
```suggestion
	return "PowerBI"
```

Reply:
Updated